tests/test_code.py

```diff
@@ -55,3 +55,12 @@ def test_RegionCode_hierarchy_attribute():
     )
 
     assert reg.hierarchy == "R5"
+
+
+def test_MetaCode_allowed_values_attribute():
```

Comment:
Not sure I see the point of this test. Creating an object directly and checking its value is something that should be covered by the python standard library. 
Reading a `MetaCode` from a file would perhaps be more useful